build.ts

```diff
@@ -1,17 +1,24 @@
 import { execSync } from "node:child_process";
 import path from "node:path";
 
-export default function setup() {
+const testbenchDir = path.join(__dirname, "vendor", "bin", "testbench");
+
+const artisan = (command: string): void => console.error(execSync(`${testbenchDir} ${command}`).toString('utf8'))
+
+export function setup(): void {
     try {
-        const testbenchDir = path.join(__dirname, "vendor", "bin", "testbench");
+        process.env.WAYFINDER_CACHE_ROUTES
+            ? artisan('route:cache')
+            : artisan('route:clear')
```

Comment:
Always clearing the routes in case the test suite failed and didn't have a chance to clean up after itself.